packages/application-extension/src/index.tsx

```diff
@@ -328,6 +364,38 @@ const mainCommands: JupyterFrontEndPlugin<void> = {
         isEnabled: () => !labShell.isEmpty('right')
       });
 
+      commands.addCommand(CommandIDs.activateSidebarWidget, {
```

Comment:
```suggestion
      commands.addCommand(CommandIDs.toggleSidebarWidget, {
```

Renaming for readability (see above).